From 84b91bcf66fc33ae1a7047751728546de42d7891 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
Date: Wed, 10 Sep 2025 23:32:09 +0200
Subject: [PATCH 1/3] package/adwaita-icon-theme: drop reference to libgtk2

As we are about to drop support for libgtk2, let's drop the reference
to libgtk2 from the adwaita-icon-theme package.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
Signed-off-by: Arnout Vandecappelle <arnout@rnout.be>
---
 package/adwaita-icon-theme/Config.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/adwaita-icon-theme/Config.in b/package/adwaita-icon-theme/Config.in
index 71b839d996..0eb856ad9e 100644
--- a/package/adwaita-icon-theme/Config.in
+++ b/package/adwaita-icon-theme/Config.in
@@ -1,6 +1,6 @@
 config BR2_PACKAGE_ADWAITA_ICON_THEME
 	bool "adwaita icon theme"
-	depends on BR2_PACKAGE_LIBGTK2 || BR2_PACKAGE_LIBGTK3
+	depends on BR2_PACKAGE_LIBGTK3
 	# host-libgtk3 -> host-librsvg -> host-pango -> host-harfbuzz
 	depends on BR2_HOST_GCC_AT_LEAST_4_9
 	help
-- 
2.43.0

